changelogs/current.yaml

```diff
@@ -1,13 +1,6 @@
-date: Pending
-
-behavior_changes:
-# *Changes that are expected to cause an incompatibility if applicable; deployment changes are likely required*
-
-minor_behavior_changes:
-# *Changes that may cause incompatibilities for some users, but should not for most*
+date: July 13, 2023
```

Comment:
I'm probably missing something but why is this change ending up in `current.yaml` and not in `1.24.9.yaml`?

Reply:
because for the 1.24.9 release `current.yaml` is the changelog file (other changelogs here have been synced from previous releases)

Reply:
Ok, but then why isn't this file "empty" (all change-log comments removed).

You understand this better than I do, so going to LGTM to unblock here.